src/PrestaShopBundle/Resources/config/routing/admin/improve/design/cms_pages.yml

```diff
@@ -0,0 +1,77 @@
+admin_cms_pages_index:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Improve\Design\CmsPage:index'
+        _legacy_controller: AdminCmsContent
+
+admin_cms_pages_search_cms_category:
+    path: /category/search
+    methods: POST
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Improve\Design\CmsPage:searchCategory'
+        _legacy_controller: AdminCmsContent
+
+admin_cms_pages_create_cms_category:
```

Comment:
you can make it shorter:

```diff
- admin_cms_pages_create_cms_category
+ admin_cms_categories_create
```

same for other routes.

Reply:
does it not break the naming convention ? My route is /cms-pages so all names should start with admin_cms_pages etc... ?